docs/docs/unit-testing.md

```diff
@@ -61,11 +61,7 @@ with a minimal config.
 
 ```js:title=jest-preprocess.js
 const babelOptions = {
-  presets: ["@babel/react", "@babel/env"],
-  plugins: [
-    "@babel/plugin-proposal-optional-chaining",
-    "@babel/plugin-proposal-class-properties",
-  ],
+  presets: ["babel-preset-gatsby"],
```

Comment:
Should installation instructions above also be changed to reflect this? Right now it says that you should install `@babel/preset-env`, `@babel/preset-react` etc., but you only have to install `babel-preset-gatsby` as far as I understand.